go/sqltypes/type.go

```diff
@@ -145,7 +145,7 @@ const (
 // bit-shift the mysql flags by two byte so we
 // can merge them with the mysql or vitess types.
 const (
-	mysqlUnsigned = 32
+	mysqlUnsigned = 128
```

Comment:
is it a problem that unsigned and binary are the same value?

Reply:
I'm having a hard time finding a reference for the binary flag, I guess they could be used mutually exclusively so it doesn't matter? Certainly seems easy enough for them to be different

Reply:
MySQL docs are difficult to look through, I updated the PR description with links to the relevant stuff.
But yeah, they shouldn't be the same value.